Utils/github_workflow_scripts/github_workflow_scripts_tests/run_docs_review_test.py

```diff
@@ -0,0 +1,24 @@
+from Utils.github_workflow_scripts.run_docs_review import pass_files_to_docs_review
+
+
+class TestsRunDocsReview:
```

Comment:
Add a test testing the fact a file that was passed this way was recognized as a release notes file (so we won't reproduce what we saw earlier)

Reply:
Not sure I completely understood your intention, but I added another test testing a RN file. 
** I'm not sure about the use of real content file names in my tests - what do you think about it?